lib/ansible/modules/network/netconf/netconf_rpc.py

```diff
@@ -0,0 +1,269 @@
+
+- name: discard changes
+  netconf_rpc:
+    request: discard-changes
+
+- name: get-schema
+  netconf_rpc:
+    xmlns: urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring
+    request:
+      get-schema:
+        identifier: ietf-netconf
+        version: "2011-06-01"
+
+- name: copy running to startup
+  netconf_rpc:
+    request:
+      copy-config:
+        source:
+          running:
+        target:
+          startup:
+
+- name: get schema list with JSON output
+  netconf_rpc:
+    request: |
+      <get>
+        <filter>
+          <netconf-state xmlns="urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring">
+            <schemas/>
+          </netconf-state>
+        </filter>
+      </get>
+    display: json
+
+- name: get schema using XML request
+  netconf_rpc:
+    request: |
+      <get-schema xmlns="urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring">
+        <identifier>ietf-netconf-monitoring</identifier>
+        <version>2010-10-04</version>
+      </get-schema>
+    display: json
+"""
+
+RETURN = """
+stdout:
+  description: The raw XML string containing configuration or state data
+               received from the underlying ncclient library.
+  returned: always apart from low-level errors (such as action plugin)
+  type: string
+  sample: '...'
+stdout_lines:
+  description: The value of stdout split into a list
+  returned: always apart from low-level errors (such as action plugin)
+  type: list
+  sample: ['...', '...']
+output:
+  description: Based on the value of display option will return either the set of
+               transformed XML to JSON format from the RPC response with type dict
+               or pretty XML string response (human-readable) or response with
+               namespace removed from XML string.
+  returned: when the display format is selected as JSON it is returned as dict type, if the
+            display format is xml or pretty pretty it is retured as a string apart from low-level
+            errors (such as action plugin).
+  type: complex
+  contains:
+    formatted_output:
+      - Contains formatted response received from remote host as per the value in display format.
+"""
+import sys
+import re
+
+try:
+    from lxml.etree import Element, SubElement, tostring, fromstring, XMLSyntaxError
+except ImportError:
+    from xml.etree.ElementTree import Element, SubElement, tostring, fromstring
+    if sys.version_info < (2, 7):
+        from xml.parsers.expat import ExpatError as XMLSyntaxError
+    else:
+        from xml.etree.ElementTree import ParseError as XMLSyntaxError
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.netconf.netconf import dispatch
+from ansible.module_utils.network.common.netconf import remove_namespaces
+
+try:
+    import jxmlease
+    HAS_JXMLEASE = True
+except ImportError:
+    HAS_JXMLEASE = False
+
+try:
+    from dicttoxml import dicttoxml
+    HAS_DICTTOXML = True
```

Comment:
Did you try using `jxmlease` for dict to xml conversion? As it is already used in other modules it makes sense to reuse it.
```
from jxmlease.dictnode import XMLDictNode
XMLDictNode({'unlock': {'target': {'candidate': None}}}).emit_xml()
```